apps/web/lib/features/task/task-input.tsx

```diff
@@ -280,6 +280,8 @@ export function TaskInput(props: Props) {
 		}
 	}, [props.autoFocus, targetEl]);
 
+	// const savedIssueType : string | null = localStorage.getItem('savedIssueType') as string && null;
```

Comment:
@JoeMakuta I think it was better to define vars like you did here (e.g. savedIssueType or lastTaskIssue), compared to directly write it like on element:

```
defaultValue={(localStorage.getItem('lastTaskIssue') as ITaskIssue) || null}
```